### PR TITLE
Document changes RELP

### PR DESCRIPTION
**Pull request by hellonearthis**
Updated the RELP to a smalled doc with just the basics including a link to Node.js RELP for a more detail.

### CODE REVIEW

**Comment by hellonearthis** [comment]
I think I got it right this time :D

File: docs/api/repl.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# REPL
+
+The `repl` provides a Read-Eval-Print-Loop (REPL) implementation, also known as an interactive toplevel. It is a simple, interactive computer programming environment that takes single user inputs (i.e. single expressions), evaluates them, and returns the result to the user.
+
+## Launching RELP
```
Comment (citycide) [inline docs/api/repl.md]: This should be **`REPL`** and not `RELP` but other than that it's a pretty good simplification.

**Comment by zcbenz** [comment]
Most of this are copied from Node, I think we should just document what's different from Node's REPL and then provide a link to it.

Also I think we can probably add a separate page to document the features of the default app, and put REPL as part of it.

**Comment by hellonearthis** [comment]
REPL doesn't work under windows last time I tried it ok.  So I feels like
stabbing in the dark as examples can't be tested.

**Comment by zcbenz** [comment]
I'm closing this pull request since most of this are copied from Node. If you are still willing to improve this please create a new pull request.

**Pull request closed without merging**